repos/system_upgrade/el7toel8/libraries/dnfplugin.py

```diff
@@ -0,0 +1,199 @@
+import contextlib
+import itertools
+import json
+import os
+import shutil
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common import guards, mounting, overlaygen, utils
+from leapp.libraries.stdlib import CalledProcessError, api, config
+from leapp.models import FilteredRpmTransactionTasks, TargetUserSpaceInfo, UsedTargetRepositories
+
+DNF_PLUGIN_NAME = 'rhel_upgrade.py'
+DNF_PLUGIN_PATH = os.path.join('/lib/python3.6/site-packages/dnf-plugins', DNF_PLUGIN_NAME)
+DNF_PLUGIN_DATA_NAME = 'dnf-plugin-data.txt'
+DNF_PLUGIN_DATA_PATH = os.path.join('/var/lib/leapp', DNF_PLUGIN_DATA_NAME)
+DNF_PLUGIN_DATA_LOG_PATH = os.path.join('/var/log/leapp', DNF_PLUGIN_DATA_NAME)
+DNF_DEBUG_DATA_PATH = '/var/log/leapp/dnf-debugdata/'
+
+
+def install(target_basedir):
+    """ Installs our plugin to the DNF plugins """
```

Comment:
Use please rather this:
```suggestion
    """
    Installs our plugin to the DNF plugins.
    """
```